### PR TITLE
Make ProgressDetails attributes public

### PR DESCRIPTION
**Pull request by marcuslinke**
Fixes issue #373

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/model/ResponseItem.java [hunk 1]
```diff
@@ -100,13 +100,25 @@ public boolean isErrorIndicated() {
         private static final long serialVersionUID = -1954994695645715264L;
 
         @JsonProperty("current")
-        long current;
+        public long current;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/ResponseItem.java]: Why do you need make it public if getter exist?

File: src/main/java/com/github/dockerjava/api/model/ResponseItem.java [hunk 2]
```diff
@@ -100,13 +100,25 @@ public boolean isErrorIndicated() {
...
 
         @JsonProperty("total")
-        long total;
+        public long total;
 
         @JsonProperty("start")
-        long start;
+        public long start;
+
+        public long getCurrent() {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/ResponseItem.java]: Can this fields be null, should they be primitive?

File: src/main/java/com/github/dockerjava/api/model/ResponseItem.java (hunk 1 above)
Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/api/model/ResponseItem.java]: You are right. Will change this.

File: src/main/java/com/github/dockerjava/api/model/ResponseItem.java (hunk 2 above)
Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/api/model/ResponseItem.java]: Targeted PR against 2.x branch, so leaving primitives. I will change to objects while merge into master.

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/ResponseItem.java]: Asking because have no idea :)

**Comment by marcuslinke** [comment]
Merged into master via 264f37b5400b318954ee56eddd4c7680ad2db924

**Pull request merged**